fesvr/elfloader.cc

```diff
@@ -15,7 +15,7 @@
 #include <vector>
 #include <map>
 
-std::map<std::string, uint64_t> load_elf(const char* fn, memif_t* memif, reg_t* entry)
+std::map<std::string, uint64_t> load_elf(const char* fn, memif_t* memif, reg_t* entry,bool ramdump)
```

Comment:
space after comma(,) before last argumet.
and maybe ramdump snapshot_mode instead of ramdump?

Reply:
OK. We will fix this as well as check similar format issues.